subsys/bluetooth/audio/Kconfig.mctl

```diff
@@ -0,0 +1,67 @@
+# Bluetooth Audio - Media control configuration options
+
+#
+# Copyright (c) 2022 Nordic Semiconductor
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# TODO: Decide, and add, top-level namespace prefix (currently none) to symbols.
+# Dependent upon where the module will be placed.
+
+config MCTL
+	bool "Support for media player control"
+	select EXPERIMENTAL
+	help
+	  Enables support for control of local and remote media players
+	  To enable support for control of a local media player, support for
+	  local media player must be enabled
+
+if MCTL
+
+config MCTL_LOCAL_PLAYER_CONTROL
+	bool "Support for control of local media player"
+	# TODO: Move MPL out of BT susbystem
+	depends on BT_MPL
+	help
+	  This option enables support for control of a local media player
+
+config MCTL_LOCAL_PLAYER_LOCAL_CONTROL
+	bool "Support for local control of local media player"
+	default y
+	depends on MCTL_LOCAL_PLAYER_CONTROL
+	help
+	  This option enables support for local application control of local
+	   media players
```

Comment:
Does it have anything with Bluetooth?

Reply:
It has to do with Bluetooth in the sense that Bluetooth is used for remote control of the local player, or for control of a remote player.
It is likely that this module may be moved out of the Bluetooth subsystem.  But that is not part of this PR.